MinPlatformPkg/PlatformInit/PlatformInitSmm/PlatformInitStandaloneMm.c

```diff
@@ -0,0 +1,153 @@
+/** @file
+  This driver will register two callbacks to call fsp's notifies.
+
+  Copyright (c) 2014 - 2016, Intel Corporation. All rights reserved.<BR>
+  Copyright (c) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include "EventNotify.h"
+#include <PiMm.h>
+
+#include <Protocol/MmReadyToLock.h>
+#include <Protocol/MmEndOfDxe.h>
+#include <Protocol/SmmReadyToBoot.h>
+#include <Protocol/SmmExitBootServices.h>
+
+#include <Library/StandaloneMmDriverEntryPoint.h>
+#include <Library/MmServicesTableLib.h>
+#include <Library/DebugLib.h>
+#include <Library/BaseMemoryLib.h>
+#include <Library/BoardInitLib.h>
+#include <Library/TestPointCheckLib.h>
+#include <Library/PerformanceLib.h>
+#include <Library/HobLib.h>
+
+EFI_STATUS
+EFIAPI
+StandaloneMmReadyToBootEventNotify (
+  IN CONST EFI_GUID  *Protocol,
+  IN VOID            *Interface,
+  IN EFI_HANDLE      Handle
+  );
+
+EFI_STATUS
+EFIAPI
+StandaloneMmReadyToLockEventNotify (
+  IN CONST EFI_GUID  *Protocol,
+  IN VOID            *Interface,
+  IN EFI_HANDLE      Handle
+  );
```

Comment:
Upon a quick glance, it seems you could just move the function implementation here (which are also in this file) and not need these prototypes. Can you check if that is possible?

Reply:
It needs these prototypes to compile.  The wrapper function thing happening here is only due to convention, but I can change it if you want.

Reply:
I talked with Kun and will be removing these wrappers as they aren't necessary.  I don't know where I got the idea that they were a convention.

Reply:
Removed the unnecessary wrappers.